docs/package.json

```diff
@@ -13,16 +13,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "axios": "^0.27.2",
     "eventsource": "^2.0.2",
-    "follow-redirects": "^1.15.1",
-    "glob-parent": "^6.0.2",
-    "got": "^12.4.1",
-    "highlight.js": "^11.6.0",
-    "markdown-it": "^13.0.1",
-    "node-forge": "^1.3.1",
-    "nth-check": "^2.1.1",
-    "url-parse": "^1.5.10",
+    "vue": "^2.6.14",
     "vuepress-theme-cosmos": "^1.0.183"
```

Comment:
@colin-axner do you have some more context about these packages? I don't think we need any of these as direct dependencies.

Reply:
Unfortunately no, these additions were the result of trying to bump the versions for these packages as they were causing "security vulnerability" notifications (nothing that affects our code, just docs - and even then I'm not sure there is anything exploitable)

I believe we get these dependencies indirectly from [vuepress-cosmos-theme](https://github.com/cosmos/vuepress-theme-cosmos)

Reply:
insert node modules heaviest matter in the universe meme